### ISSUE TITLE
Firefox Devtools Instructions 

### ISSUE THREAD

**Issue by arky** [OpenTechSchool/js-beginners-1#24]
Would love to use this to introduce JavaScript programming. However the tutorial only mentions Chrome but not Firefox Devtools. 

Thanks @xMartin for pointing me to this resource.

**Comment by xMartin** [comment]
A bit more background:

Arky works for Mozilla and cares for local communities and as part of this education.

The question is, if we can/want to integrate mentioning and explanation of Firefox and the Devtools in addition to Chrome. On the one hand I don't like the focus on that one product (Google Chrome), on the other hand we did this to keep it simple. I always tell people that there are other great tools around, this is just an example. I would be open for proposals how to add Firefox and make the explanations more open and vendor-independant.

Same issue maybe with the editor (Sublime Text) for day 2.

Maybe this is a sign of growth, that we have to take more care now about those kind of issues as our user base gets more diverse.

**Comment by theophani** [comment]
I agree with everything you’ve said, @xMartin. I kept it to Chrome and Sublime Text 2 for the first event to simply things in that first workshop. Now is a good time to remove those specifics.

**Comment by gnunicorn** [comment]
Totally agree. Though removing the reference of tools kinda makes it hard and artificial.

So, I was wondering if it was feasible to have each block that is tool-specific available for each tool. At the beginning the learner selects the tools they use or want to see the stuff in (like chrome+sublime or firefox+vim) and then the parts using other tools are hidden throughout the material. I don't know if that is feasible and doesn't add too much confusion, but maybe it is a solution worth trying ...

Extra Plus: if you could click to select the same paragraph for a different tool to allow learners to get some idea, how it might be done in other tools... Diversity and stuff, yo know.

( @arky welcome in the discussion. Not sure if you remember, but we have met over two years ago at the Mozilla Breakfast Meetup at St. Oberholz, when you were in Berlin. Really happy to see you on board.)

**Comment by arky** [comment]
Thanks for the feedback. Yes, I remember you @ligthyear. Still wear that cool unhosted t-shirt you gave me in Berlin.

**Comment by gabeno** [comment]
Hello everyone, this is an important concern. I concur with @ligthyear's suggestion that for the same content a learner may be pointed to how they could do it using other tools to accomodate the potentially diverse user base. This also encourages learners to realize that there are other tools to use do the achieve the same purpose and it depends on a matter of preferences.

Ideally, a learner starts their journey with a given set of tools and they may change them over time as they advance their skills.
